pom.xml

```diff
@@ -212,7 +212,7 @@ under the License.
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
       <version>${testng.version}</version>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
testng wasn't compiling, so I altered as per the comment here:
https://github.com/cbeust/testng/issues/1458

Reply:
I am not sure about this fix
I recall having the same issue however they always prefer the pom file this way - not sure what is the reason

Reply:
As long as we can provide a compilation line for the artifact users, I have no issue leaving it as is. I was unable to work out what the compilation line is with the scope as test. (Also google showed that I'm not the only one with this specific problem).

If we could change it and then change it back that would also work, as I can then point the artifact users to a specific commit id.

Reply:
Let's try to figure out what the problem is as this fix may be hiding some other issue.  TestNG should work under test scope.  For example, launching a characterization Job is a normal runtime job and TestNG may not be in the dependency path.  When running "Test" code in the test path as "mvn test", Maven will recognize that TestNG needs to be in the dependency path.  But I suspect you already know this.    The only reason I can think of that you need the scope to be "compile" would be somehow your main code is attempting to execute TestNG.  Perhaps if you could be more explicit as to what the problem is so I can reproduce it!

Reply:
The way I produced the issue was pulling the memory and Java repos, switching to 1.1.X branch, and then compiling all 3 repos using maven build without tests.